### PR TITLE
Update ex11_31.cpp

### PR DESCRIPTION
**Pull request by qwert2603**

### CODE REVIEW

File: ch11/ex11_31.cpp [hunk 1]
```diff
@@ -29,17 +29,17 @@ int main()
...
-    while (count) {
-        if (found->second == work) {
-            authors.erase(found);
-            break;   
-        }
-        ++found;
-        --count;
+    // using function equal_range to get range itertors
+    auto a = authors.equal_range(author);
+    while(a.first != a.second)
```
Comment (pezy) [inline ch11/ex11_31.cpp]: note requirement of the question:

> Use **find** to find **an element** in the multimap and erase that element

File: ch11/ex11_31.cpp [hunk 2]
```diff
@@ -29,17 +29,17 @@ int main()
...
+    {
+        if(a.first->second == work) // if it is required element,
+            a.first == authors.erase(a.first);  // delete element and move iterator to the next element.
```
Comment (Mooophy) [inline ch11/ex11_31.cpp]:
```cpp
a.first == authors.erase(a.first);
```
Is `==` really what you want? I guess you want an assignment `=` here..

**Comment by Mooophy** [comment]
The variable name `a` is another problem, I just don't think it's a good name, quite hard to figure out what it stands for..

**Comment by Mooophy** [comment]
@qwert2603 
Sorry for that we can't accept your pull request. Still looking forward more contribution from you.
Good luck.

**Pull request closed without merging**